### DISCUSSION TITLE
AMLL Player设置背景问题

### DISCUSSION THREAD

**Discussion by HickoryTrail** [amll-dev/applemusic-like-lyrics#230] [Q&A]
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/80823417/85b3139c-89f7-485f-9bc8-a238f2d3cae3)
在最新版AMLL Player外挂播放器中，设置界面背景是黑色的，文字也是黑色的，什么也看不到

**Comment by Linho1219** [comment]
在最新版 AMLL Player 中未能复现，推测已经修复。

因时间久远，现关闭此讨论。如仍遇相关问题请重新打开或新建条目，敬请谅解。
